RecoTauTag/RecoTau/interface/DeepTauBase.h

```diff
@@ -57,9 +62,10 @@ namespace deep_tau {
 
   class DeepTauBase : public edm::stream::EDProducer<edm::GlobalCache<DeepTauCache>> {
   public:
-    using TauType = pat::Tau;
+    using TauType = edm::View<reco::BaseTau>;
     using TauDiscriminator = reco::TauDiscriminatorContainer;
-    using TauCollection = std::vector<TauType>;
+    using TauCollection = TauType;
```

Comment:
I think you can omit TauType and directly write `using TauCollection = edm::View<reco::BaseTau>;`